tools/dotnet-fssrgen/dotnet-fssrgen.xproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
remove that file, see https://github.com/fsharp/FSharp.Compiler.Service/pull/556#issuecomment-212895651